packages/Link/Link.jsx

```diff
@@ -25,13 +46,16 @@ const Link = ({ reactRouterLinkComponent, invert, children, ...rest }, context)
     warn('Link', 'The props `reactRouterLinkComponent` and `to` must be used together.')
   }
 
-  return React.createElement(
-    reactRouterLinkComponent || 'a',
-    {
-      ...safeRest(rest),
-      className: getClassName(invert, context),
-    },
-    children
+  return (
+    <StyledLink
+      {...safeRest(rest)}
+      as={reactRouterLinkComponent || 'a'}
+      invert={invert}
+      context={context}
+      data-testid="link"
```

Comment:
Let's put this prop before `safeRest(rest)` that way it can be overridden since is not essential to its consumption. Either this or stripping `data-testid` on build should be established as a practice.

Reply:
removed `data-testid` since it's not actually being used in the tests anymore